shipyard_blueprints/hubspot/shipyard_hubspot/hubspot.py

```diff
@@ -85,3 +102,184 @@ def connect(self):
         else:
             self.logger.info("Successfully connected to Hubspot API")
             return 0
+
+    def export_data(self, export_type: str, **kwargs):
+        export_type = validate_export_type(export_type)
+        self.logger.debug("Exporting data from Hubspot API")
+        if export_type == "list":
+            return self.export_list(**kwargs)
+        # elif export_type == "view":
```

Comment:
Opted out of including view export since there is no easy way to have this in a user friendly format in our UI. 
it takes search conditions as an array where as the list can be premade in hubspot and referred to in the blueprint.